_atoi.c

```diff
@@ -0,0 +1,32 @@
+#include "main.h"
+
+/**
+*_atoi - Takes a string as argument and returns its value of type int
+*@s: The number in string form
+*Return: an integer
+*/
+
+int _atoi( char *s)
+{
+
+unsigned int res = 0;
+int sign = 1;
+
+if (*s == '\0')
+return (0);
+
+do {
+
+if (*s == '-')
+sign *= -1;
+
+else if (*s >= '0' && *s <= '9')
+flag = 1;
+res = (res * 10) + (*s - '0');
+
+else if (res > 0)
+break;
+} while (*s++);
+
+return (res * sign);
+}
```

Comment:
@nyabutovalentine18 , please remove this file on this PR , i want to see only the readme on this PR .
to do that : 
- In your computer , delete this file 
- After that : 
- git add .
- git commit -m 'remove all .c file '
- Push again those modifications .